cmd/antrea-agent/agent.go

```diff
@@ -142,6 +142,13 @@ func run(o *Options) error {
 	enableMulticlusterGW := features.DefaultFeatureGate.Enabled(features.Multicluster) && o.config.Multicluster.EnableGateway
 	enableMulticlusterNP := features.DefaultFeatureGate.Enabled(features.Multicluster) && o.config.Multicluster.EnableStretchedNetworkPolicy
 	enableFLowExporter := features.DefaultFeatureGate.Enabled(features.FlowExporter) && o.config.FlowExporter.Enable
+	enableAntreaProxy := *o.config.AntreaProxy.Enable
+	// TODO (lhongliang): remove this in the future. This is for compatibility with AntreaProxy disabled.
```

Comment:
I think it's safe to omit the TODO: when the Feature gate is removed from the code, we will automatically remove this code (or the build will fail).

Reply:
Removed this.